docs/graalvm-m1-mac.md

```diff
@@ -0,0 +1,45 @@
+## Installing GraalVM on M1 Mac
+
+This guide describes how to setup the specific version of GraalVM on M1 Mac
+using [`sdkman`](https://sdkman.io/). You can also use
+[`jenv`](https://www.jenv.be/) or setup `PATH` and `JAVA_HOME` environment
+variables manually.
```

Comment:
So the whole instruction below (except downloading and extracting the package) is just to avoid setting two environment variables manually?

Reply:
The purpose of the instruction is to provide an alternative to `sdk install java 21.1.0.r11-grl` command, which will work on any other platform, but won't work on M1 Macs. I reworded this.